### PR TITLE
Update qe66_archer2_gnu.md

### PR DESCRIPTION
**Pull request by jsindt**
Updating to current instructions (now with proper HDF5 and BLAS/LAPACK/SCALAPACK linking)

### CODE REVIEW

File: QuantumEspresso/qe66_archer2_gnu.md [hunk 1]
```diff
@@ -4,23 +4,35 @@ Build Instructions for QE 6.6 on ARCHER2
...
 export CC=cc
 export FC=ftn
 export F77=ftn
 export F90=ftn
-./configure --prefix=${PATH_TO}/q-e-qe-6.6
+
+export BLAS_LIBS=/opt/cray/pe/libsci/20.10.1.2/gnu/9.1/x86_64/lib/libsci_gnu_mpi.a
+export LAPACK_LIBS=/opt/cray/pe/libsci/20.10.1.2/gnu/9.1/x86_64/lib/libsci_gnu_mpi.a
+export SCALAPACK_LIBS=/opt/cray/pe/libsci/20.10.1.2/gnu/9.1/x86_64/lib/libsci_gnu_mpi.a
+export FFT_LIBS=/opt/cray/pe/fftw/3.3.8.8/x86_rome/lib/libfftw3_mpi.a
```
Comment (aturner-epcc) [inline QuantumEspresso/qe66_archer2_gnu.md:22]: Should these not be empty if you are using the compiler wrapper scripts as they include the libraries by default? i.e.:

```
export BLAS_LIBS=
export LAPACK_LIBS=
export SCALAPACK_LIBS=
export FFT_LIBS=
```

If the build tool does not like them being completely empty, you can set them all to a single space " ". Or, is there something in the QE build that means that these have to be explicitly specified?

**Review by aturner-epcc — CHANGES_REQUESTED** [review]
Just a question on setting explicit paths.

File: QuantumEspresso/qe66_archer2_gnu.md (hunk 1 above)
Comment (jsindt) [reply QuantumEspresso/qe66_archer2_gnu.md]: I hadn't thought of that but have now tested and setting them all to a single space works (if empty, QE will try to use its own version of these and things go bad). I've changed the instructions

**Review by aturner-epcc — APPROVED** [review]
LGTM

**Pull request merged**